express-101/src/index.js

```diff
@@ -2,8 +2,11 @@ const express = require('express');
 
 const app = express();
 
-// 1) Create Routes
-
-// 2) Start server on port 3000
-
-module.exports = app;
+app.get('/', (req, res) => res.send('Hey, I am server response'));
+app.get('/movie/:title', (req, res) => {
+    const { title } = req.params;
+    res.send(`My favorite movie is ${title}`);
+});
+
+const port = process.env.PORT || 3000;
```

Comment:
Hi @prateekinstillion , could you clarify what's the benefit of using this construct as opposed to plain `const port = 3000;`. I'd like to know your thoughts here.

Reply:
Hi @abhi18av, regarding the context of problem it is not mandatory but when the application server is to deployed in a live production environment, your server runs on a different variable port denoted by `process.env.PORT`. I hope I made it clear. :)